docfx_project/ziti/clients/android.md

```diff
@@ -1,3 +1,3 @@
 ## Android
 
-The Android tunneler is currently in an open beta. Find the app in the [Play store here](https://play.google.com/store/apps/details?id=io.netfoundry.ziti.tunnel)
\ No newline at end of file
+The Android tunneler is currently in an open beta. Find the app in the [Play store here](https://play.google.com/store/apps/details?id=io.netfoundry.ziti.tunnel). The source code for [the tunneler is here in GitHub](https://github.com/openziti/ziti-tunnel-android/) and it is based on [the edge SDK for Java/Kotlin](https://github.com/openziti/ziti-sdk-jvm).

```

Comment:
"the tunneler is here in GitHub" --> "the Android tunneler is in GitHub". 
"the edge SDK for Java/Kotlin" --> "the ziti-sdk-jvm SDK" or "the SDK for the JVM" but not "Java/Kotlin". Java/Kotlin are only two languages for the JVM.